docs/machine-learning/tutorials/object-detection-model-builder.md

```diff
@@ -0,0 +1,263 @@
+---
+title: 'Tutorial: Detect objects in images with Model Builder'
+description: This tutorial illustrates how to build an object detection model using ML.NET Model Builder and Azure ML to detect stop signs in images.
+author: briacht
+ms.author: brachtma
+ms.date: 03/12/2021
+ms.topic: tutorial
+ms.custom: mlnet-tooling
+#Customer intent: As a non-developer, I want to use Model Builder to automatically generate a model to detect stop signs in images using Model Builder.
+---
+
+# Tutorial: Detect stop signs in images with Model Builder
+
+Learn how to build an object detection model using ML.NET Model Builder and Azure ML to detect and locate stop signs in images.
+
+In this tutorial, you learn how to:
+
+> [!div class="checklist"]
+>
+> - Prepare and understand the data
+> - Choose the scenario
+> - Choose the training environment
+> - Load the data
+> - Train the model
+> - Evaluate the model
+> - Use the model for predictions
+
+> [!NOTE]
+> Model Builder is currently in Preview.
+
+## Prerequisites
+
+For a list of prerequisites and installation instructions, visit the [Model Builder installation guide](../how-to-guides/install-model-builder.md).
+
+## Model Builder object detection overview
+
+Object detection is a computer vision problem. While closely related to image classification, object detection performs image classification at a more granular scale. Object detection both locates _and_ categorizes entities within images. Use object detection when images contain multiple objects of different types.
+
+![Screenshots showing Image Classification versus Object Classification.](./media/object-detection-onnx/img-classification-obj-detection.png)
```

Comment:
While this may render correctly on the live docs site, it doesn't render on GitHub due to the file name extension being `PNG` instead of `png`. This is probably because the site runs on Windows, while GitHub runs on Linux.

Reply:
Fixed: https://github.com/dotnet/docs/pull/23300/commits/a4bb502948fa92751c1c6a5ad4bb36946ad90453